eng/config/labels.ts

```diff
@@ -32,6 +32,10 @@ export const AreaLabels = defineLabels({
     color: "846da1",
     description: "Issues for Azure specific ide features",
   },
+  "cli/psh": {
```

Comment:
Also need to add a record correlating the label with a path (or no path) in the AreaPaths structure on line 44 below

Reply:
Probably no path, in this case